sql/src/main/java/io/crate/breaker/RowAccounting.java

```diff
@@ -41,6 +42,13 @@ public RowAccounting(Collection<? extends DataType> columnTypes, RamAccountingCo
         this.ramAccountingContext = ramAccountingContext;
     }
 
+    public RowAccounting(Collection<? extends DataType> columnTypes,
+                         RamAccountingContext ramAccountingContext,
+                         int extraSizePerRow) {
```

Comment:
can you please provide some more context around what `extraSizePerRow` is (maybe even the array example). being a public constructor I think it'd be helpfull